src/components/atoms/AtomRadioButton.vue

```diff
@@ -17,12 +17,16 @@
 <script>
 export default {
     name: 'AtomRadioButton',
-    data() {
-        return {
-            radio: '',
-        };
-    },
     props: {
+        /**
+         * selected radio option
+         * @selectedRadio in string
+         */
+        currentSelectedRadio: {
+            type: String,
+            required: true,
+            default: null,
```

Comment:
when required is true, do we need default value?

Reply:
gives warning but not break the code.